packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2585,10 +2585,11 @@ class BuildOwner {
   /// the [FocusScopeNode] for a given [BuildContext].
   ///
   /// See [FocusManager] for more details.
-  FocusManager focusManager = FocusManager();
+  FocusManager focusManager;
```

Comment:
should/could this be final?

Reply:
It's non-final now because it's mutated in the test widgets flutter binding.  It looks like it's done to reset the global pointer routes and such...

Reply:
dubious, but that's an issue for another PR... :-)

Reply:
Agreed.  There are a few follow-on PRs that I want to put together now that I've been there poking around.